investor_wallet_platform_base/security/iwp_security.xml

```diff
@@ -305,6 +305,17 @@
 
 		<!-- Mail Templates -->
 
+	    <record id="mail_template_emc_user_read_mail_template_rule" model="ir.rule">
+	        <field name="name">mail_template_emc_manager_own_loan_issue_line_rule</field>

```

Comment:
manager appears in the name of the rule while it's defined for user.

Reply:
appart this the rest LGTM

Reply:
good catch, fixed.